CareKit/CareCard/OCKCareCardTableViewCell.h

```diff
@@ -47,6 +48,10 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (void)careCardTableViewCell:(OCKCareCardTableViewCell *)cell didSelectInterventionActivity:(OCKCarePlanActivity *)activity;
 
+- (nullable NSString *)careCardTableViewCell:(OCKCareCardTableViewCell *)cell selectedStateTextForInterventionActivity: (OCKCarePlanEvent *)interventionActivity atIndex:(NSInteger) index;
```

Comment:
parameter name 'interventionActivity' of type OCKCarePlanEvent might be misleading.